src/main/resources/fxml/MainScreenView.fxml

```diff
@@ -0,0 +1,33 @@
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ProgressBar?>
+<?import javafx.scene.control.TableColumn?>
+<?import javafx.scene.control.TableView?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.web.WebView?>
+<HBox xmlns:fx="http://javafx.com/fxml" fx:controller="com.webonise.controllers.MainScreenController" spacing="8">
+    <padding>
+        <Insets left="5.0" right="5.0" top="5.0" bottom="5.0"/>
+    </padding>
```

Comment:
Remove padding, spacing and height width properties from FXML and use seprate css for that. We dont use inline css into FXML file.